arch/x86/deflate_quick.c

```diff
@@ -92,11 +92,11 @@ static inline long compare258(const unsigned char *const src0, const unsigned ch
         "cmp        $256 + 16, %[ax]\n\t"
         "jb         1b\n\t"
 
-#ifdef X86
+# if !defined(__x86_64)
         "movzwl     -16(%[src0], %[ax]), %[dx]\n\t"
-#else
+# else
         "movzwq     -16(%[src0], %[ax]), %[dx]\n\t"
-#endif
+# endif
```

Comment:
I see no reason to change the whitespace here.
The whitespace makes it easier to see that this if is nested inside another if.

Reply:
> The whitespace makes it easier to see that this if is nested inside another if.

Yes, this is the case: `#if !defined(__x86_64) #else #endif` is nested within the `#else` clause of `#ifdef _MSC_VER`.

Reply:
I must have been confused, I could swear I thought you were removing the spaces. All good then.